package.json

```diff
@@ -67,21 +63,19 @@
     "jest-fetch-mock": "~2.1.1",
     "jest-junit": "~6.3.0",
     "jest-transform-graphql": "~2.1.0",
-    "keypress": "~0.2.1",
     "lerna": "~3.13.0",
     "lodash.debounce": "~4.0.8",
-    "multispinner": "~0.2.1",
     "prettier": "~1.16.4",
     "prettier-check": "~2.0.0",
-    "stream-snitch": "~0.0.3",
+    "rimraf": "~2.6.3",
     "wait-for-expect": "~1.1.0"
   },
   "optionalDependencies": {
     "bundlesize": "~0.17.1",
     "sharp": "~0.22.1"
   },
   "engines": {
-    "node": "8.10.0 || >=10.7.0",
```

Comment:
Is node 8 no longer necessary for now.sh?

Reply:
It is not! :tada: